### ISSUE TITLE
Running winget through the app execution alias gives no output

### ISSUE THREAD

**Issue by ItsPizzaTime1501** [microsoft/winget-cli#2025]
### Brief description of your issue

Running winget from the app execution alias located in `%LOCALAPPDATA%\Microsoft\WindowsApps` gives no console output. It still works fine if I run it from `C:\Program Files\WindowsApps\Microsoft.DesktopAppInstaller_1.18.431.0_x64__8wekyb3d8bbwe` however I get no console output if I run it from the app execution alias. I've already tried stable versions and older versions and disabling and re-enabling the alias with no luck. I checked logs and running it from the alias doesn't create a log.

This may be more of a problem with windows than with winget, not sure.

### Steps to reproduce

Just happened randomly, sometimes a reboot or re-install fixes it other times it doesn't. 

### Expected behavior

I should get console output from running `winget`

### Actual behavior

I do not console output from running `winget`

### Environment

```shell
Windows Package Manager (Preview) v1.3.0-preview
Copyright (c) Microsoft Corporation. All rights reserved.

Windows: Windows.Desktop v10.0.19044.1586

Logs: %TEMP%\WinGet
```

**Comment by jedieaston** [comment]
Do other App Execution Aliases work when winget breaks? (If you have Windows Terminal installed, `wt.exe` is a alias to launch it)

**Comment by ItsPizzaTime1501** [comment]
> Do other App Execution Aliases work when winget breaks? (If you have Windows Terminal installed, `wt.exe` is a alias to launch it)

Yeah Windows Terminal works fine (only other one I have). Seems to be an issue with only winget.

**Comment by ItsPizzaTime1501** [comment]
So this is like the most random thing ever but it seems to only happen if I transfer files over IRC with HexChat. I don't know how or why but that seems to be what's causing this to happen although I'd have no clue how the two are linked.
